bm_serial_messages.h

```diff
@@ -125,12 +131,86 @@ typedef struct {
 } __attribute__ ((packed)) bm_serial_dfu_finish_t;
 
 typedef struct {
-  // Node id 
+  // Node id
   uint64_t node_id;
   // Reboot Reason
   uint32_t reboot_reason;
-  // git hash 
+  // git hash
   uint32_t gitSHA;
-  // reboot count 
+  // reboot count
   uint32_t reboot_count;
 } __attribute__ ((packed)) bm_serial_reboot_info_t;
+
+typedef struct {
+  // Node ID of the target node for which the request is being made. (Zeroed = all nodes)
+  uint64_t target_node_id;
+} __attribute__((packed)) bm_serial_device_info_request_t;
+
+typedef struct {
+  // Node ID of the responding node
+  uint64_t node_id;
+
+  // Vendor ID of the hardware module implementing the BM Node functions
+  uint16_t vendor_id;
+
+  // Product ID for the hardware module implementing the BM Node functions
+  uint16_t product_id;
+
+  // Factory-flashed unique serial number
+  uint8_t serial_num[16];
```

Comment:
Note here that once we have products that are not ST, we may want to revist this fixed length and possibly other things that define "device info".